src/database/prisma/migrations/20230109132332_init/migration.sql

```diff
@@ -0,0 +1,56 @@
+-- CreateTable
```

Comment:
Maybe we should add prisma/migrations folder to gitignore

Reply:
added